### PR TITLE
Added function to load table from csv file, and applied to CGAC table

### PR DESCRIPTION
**Pull request by jworcestBAH**
This adds a function to load tables from csv files, the CGAC table, and the file to load CGACs from

### CODE REVIEW

**Comment by jworcestBAH** [comment]
Hold off on this for a few minutes, I'm testing one more thing

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
FYI @jworcestBAH: link to Jenkins is stuck on an obsolete address, so here's the failing build output:
Traceback (most recent call last):
  File "runTests.py", line 5, in <module>
    from jobTests import JobTests
  File "/backend/tests/jobTests.py", line 4, in <module>
    from baseTestValidator import BaseTestValidator
  File "/backend/tests/baseTestValidator.py", line 9, in <module>
    from dataactvalidator.app import createApp
  File "/backend/dataactvalidator/app.py", line 11, in <module>
    from dataactvalidator.validation_handlers.validationManager import ValidationManager
  File "/backend/dataactvalidator/validation_handlers/validationManager.py", line 16, in <module>
    from dataactvalidator.validation_handlers.validator import Validator
  File "/backend/dataactvalidator/validation_handlers/validator.py", line 6, in <module>
    from dataactcore.models.validationModels import TASLookup
ImportError: cannot import name TASLookup

**Comment by jworcestBAH** [comment]
Thanks @klrBAH, I've fixed that now.

File: dataactcore/migrations/versions/3a79d7a4d1c1_merge_rule_tables.py [hunk 1]
```diff
@@ -69,15 +69,15 @@ def upgrade_validation():
...
+    op.create_foreign_key('fk_file_id', 'rule', 'file_type', ['file_id'], ['file_id'])
     op.create_foreign_key('fk_target_file_id', 'rule', 'file_type', ['target_file_id'], ['file_id'])
     ### end Alembic commands ###
 
 
 def downgrade_validation():
     ### commands auto generated by Alembic - please adjust! ###
     op.drop_constraint('fk_target_file_id', 'rule', type_='foreignkey')
-    op.drop_constraint(None, 'rule', type_='foreignkey')
+    #op.drop_constraint('fk_file_id', 'rule', type_='foreignkey')
```
Comment (nmonga91) [inline dataactcore/migrations/versions/3a79d7a4d1c1_merge_rule_tables.py]: Why is this line commented out?

**Comment by nmonga91** [comment]
Everything else looks good. Waiting for response on previous comment to merge.

File: dataactcore/migrations/versions/3a79d7a4d1c1_merge_rule_tables.py (hunk 1 above)
Comment (jworcestBAH) [inline dataactcore/migrations/versions/3a79d7a4d1c1_merge_rule_tables.py]: Thanks, this was a temporary thing due to naming this constraint while my DB was on a later version.  Uncommented it now.

**Comment by nmonga91** [comment]
Everything looks good, merging.

**Pull request merged**